packages/terra-overlay/src/Overlay.scss

```diff
@@ -0,0 +1,28 @@
+.terra-Overlay {
+  background-color: $terra-dark;
+  height: 100%;
+  left: 0;
```

Comment:
overlay will be either fullscreen or fill the entirety of a DOM element (width and height = 100%), so it work in bi-di. Attached a gif above of rtl